### PR TITLE
Lab13

### PR DESCRIPTION
**Pull request by Ostyantic**

### CODE REVIEW

File: JS/app.js [hunk 1]
```diff
@@ -19,6 +19,14 @@ let uniqueProducts = [];
 // setting a counter to keep track of number of clicks
 let clickCounter = 0;
 
+// on page load: loading users click counter from LS with an if statement. if the users clicks are less than or equal to 24, the clicks from LS will be loaded and the user can continue. If clicks exceed 24, this if statement is ignored and users click counter starts at 0.
+if (localStorage.getItem('userClicks') <= 24) {
+  let loadedClickCounter = localStorage.getItem('userClicks');
+  clickCounter = JSON.parse(loadedClickCounter);
+} else {
+  localStorage.removeItem('products');
+}
```
Comment (Barnord) [inline JS/app.js:28]: Very clean piece of code here!

File: JS/app.js [hunk 2]
```diff
@@ -130,6 +145,23 @@ function handleClickedProduct(event) {
...
+
+  // declaring new variable and assigning it a stringified productsArr and console logging for testing
+  let productsArrConvertedForLS = JSON.stringify(productsArr);
+  let clickCounterConvertedForLS = JSON.stringify(clickCounter);
+  // console.log(productsArrConvertedForLS);
+  // console.log(clickCounterConvertedForLS);
+
+  // setting productsArr into local storage with the name products as the key and console logging for testing
+  localStorage.setItem('products', productsArrConvertedForLS);
+  localStorage.setItem('userClicks', clickCounterConvertedForLS);
```
Comment (Barnord) [inline JS/app.js:161]: I always like seeing the saving to local storage done in the event handler. Great work!

**Review by Barnord — COMMENTED** [review]
Excellent work! Once you get rid of the commented out code, this will be very clean!

File: JS/app.js (hunk 1 above)
Comment (Ostyantic) [reply JS/app.js:28]: Thank you! Thinking back on it, this one was really fun. figuring out how to load the click counter while also clearing out the saved data at max clicks felt as if i was piecing a puzzle together.

**Pull request merged**